kernel/comps/virtio/src/device/block/mod.rs

```diff
@@ -119,15 +119,11 @@ impl VirtioBlockConfig {
         SECTOR_SIZE
     }
 
-    pub(self) fn block_size(&self) -> usize {
-        self.blk_size as usize
+    pub(self) fn block_size(this: &SafePtr<Self, IoMem>) -> usize {
+        field_ptr!(this, Self, blk_size).read_once().unwrap() as usize
```

Comment:
Let's call this method `read_block_size` to emphasize the fact that it is not a simple getter.